product-matrix.json

```diff
@@ -1,47 +1,57 @@
 {
   "list": [
     {
-      "comments": "IntelliJ 2018.3, AS 3.4",
-      "name": "Android Studio 3.4",
-      "version": "3.4",
+      "comments": "IntelliJ 2018.3, AS 3.4.1",
+      "name": "Android Studio 3.4.1",
+      "version": "3.4.1",
       "ijVersion": "2018.3",
       "ideaProduct": "android-studio",
-      "ideaVersion": "183.5370308",
-      "dartPluginVersion": "183.5901",
-      "sinceBuild": "183.5153",
+      "ideaVersion": "183.5522156",
+      "dartPluginVersion": "183.6270",
+      "sinceBuild": "183.6156",
```

Comment:
Unrelated to this PR, it might be nice to generate a simple markdown file, derived from the info in `product-matrix.json`, that contains the supported API version ranges, one per line. That would make it easier to compare which products supported what version ranges.